cypress/e2e/app_tests/test_1.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('first test suite', () => {
+
+    context('Check Step component', () => {
+        it('Verify section title and buttons are clickable', () => {
+            cy.visit('/pages/layout/stepper');
+            cy.get('.horizontal .step-content h3.ng-star-inserted').should('contain.text', 'Step content #1')
+            cy.get('.horizontal .ng-star-inserted.appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions').eq(1).click()
+
+            cy.get('.horizontal .step-content h3.ng-star-inserted').should('contain.text', 'Step content #2')
+            cy.get('.horizontal .step-content .ng-star-inserted.appearance-filled.size-medium.status-primary.shape-rectangle.transitions.ng-star-inserted').eq(1).click() 
+
+            cy.get('.horizontal .step-content h3.ng-star-inserted').should('contain.text', 'Step content #3')
+            cy.get('.horizontal .step-content .ng-star-inserted.appearance-filled.size-medium.status-primary.shape-rectangle.transitions.ng-star-inserted').eq(1).click() 
+
+            cy.get('.horizontal .step-content h3.ng-star-inserted').should('contain.text', 'Step content #4')
+            cy.get('.horizontal .step-content .ng-star-inserted.appearance-filled.size-medium.status-primary.shape-rectangle.transitions.ng-star-inserted').eq(0).click() 
```

Comment:
I can two ways to improve your code:
1. using aliases:
```
            const testData=['Step content #1','Step content #2', 'Step content #3', 'Step content #4']
            cy.get('.horizontal .step-content h3.ng-star-inserted').as('header')
            cy.get('.horizontal .ng-star-inserted.appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions').eq(1).as("next");

            cy.get('@header').should('contain.text', testData[0])
            cy.get('@next').click();
            cy.get('@header').should('contain.text', testData[1])
            cy.get('@next').click();
            cy.get('@header').should('contain.text', testData[2])
            cy.get('@next').click();
            cy.get('@header').should('contain.text', testData[3])
```

2. using loop
 ```
           const testData = ['Step content #1', 'Step content #2', 'Step content #3', 'Step content #4']
            cy.get('.horizontal .step-content h3.ng-star-inserted').as('header')
            cy.get('.horizontal .ng-star-inserted.appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions').eq(1).as("next");
            for (let index = 0; index < testData.length; index++) {
                cy.get('@header').should('contain.text', testData[index])
                index !== 3 ? cy.get('@next').click() : "Finish"
            }
```

Reply:
Thanks. Tried both and selected loops